backend/app/routers/keycloak.py

```diff
@@ -32,7 +33,13 @@ async def register() -> RedirectResponse:
 @router.get("/login")
 async def login() -> RedirectResponse:
     """Redirect to keycloak login page."""
-    return RedirectResponse(settings.auth_url)
+    return RedirectResponse(
+            keycloak_openid.auth_url(
+                redirect_uri=settings.auth_redirect_uri,
+                scope="openid email",
```

Comment:
If there are additional values that need to be grabbed from the `userinfo` endpoint, there could be need for it. It might be a  better solution to keep the scopes static though, and instead create a scope in keycloak that contains that info. (i.e. create a `clowder2` scope for the client and then map the need values to it, and then keep it static like `scope="openid clowder2"`)

Maybe adding a `scopes` variable in `keycloak_auth` would be a better middle ground solution?